src/api/content-rest-api/api/nodes.api.ts

```diff
@@ -1980,4 +1737,21 @@ parameter are returned in addition to those specified in the **fields** paramete
             contentTypes, accepts , NodeEntry);
     }
 
+    getDirectAccessUrl(nodeId: string): Promise<DirectAccessUrlEntry> {
```

Comment:
Minor: Is it ok to use "get" for POST urls ? 

It might be ... maybe  other examples, eg. /search API or others.

Reply:
Yes good point. I can change it to "requestDirectAccessUrl" to be more in line with the endpoint name